### ISSUE TITLE
Missing Checkbox PropertyFilter + can't set property name

### ISSUE THREAD

**Issue by DenisD3D** [seratch/notion-sdk-jvm#23]
Hi,
I found that the checkbox entry was missing in notion.api.v1.model.common. Adding 
```
@SerializedName("checkbox") Checkbox("checkbox");
```
should solve this part of the issue.

Also I can't found how to set the name of the property :
My checkbox is called Done in notion
if i managed to make the following code where i can't set the property name
```
println(notionClient.queryDatabase(databaseId = database_id, filter = PropertyFilter(PropertyType.Checkbox (Missing as of now), checkbox = CheckboxFilter("true"))))
```
I end wit this error ; `notion.api.v1.exception.NotionAPIError: Got an error from Notion (status: 400, code: validation_error, message: Could not find property with name or id: checkbox)`
I suppose it take the property type as name

Additionnal thing but not important : 
CheckboxFilter equals take a String => Shouldn't it be a bool ? 

Thanks
Denis

**Comment by seratch** [comment]
Thanks! Fixed

**Comment by seratch** [comment]
The fix is included in v0.1.15

**Comment by DenisD3D** [comment]
Does this add the ability to set the property name?

**Comment by seratch** [comment]
@DenisD3D Thanks for your reply. As the unit tests use "title" for it, the tests haven't detect my mistakes on the property type 🤦 I just released v0.1.16 including the fix for it (the jar files will be available in the Maven Central repos soon).
